### DISCUSSION TITLE
Restarting Lazer resets your skin to the default skin

### DISCUSSION THREAD

**Discussion by minasminas** [ppy/osu#15958] [Q&A] [answered]
As the title says, selecting a skin, ending the game and starting it again resets the skin to the osu!lazer default skin.

Logs:
[performance.log](https://github.com/ppy/osu/files/7662337/performance.log)
[runtime.log](https://github.com/ppy/osu/files/7662339/runtime.log)
[updater.log](https://github.com/ppy/osu/files/7662340/updater.log)
[network.log](https://github.com/ppy/osu/files/7662341/network.log)

**Comment by bdach** [comment, marked as answer]
Addressed by #15959
